examples/executable-prompts/README.md

```
@@ -0,0 +1,60 @@
+# Executable Prompts Example
+
+This example demonstrates how to use executable scripts and binaries as prompt generators in promptfoo.
+
+To run it:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)